### ISSUE TITLE
Images on Git Repository

### ISSUE THREAD

**Issue by eduAbreu** [thedevdojo/voyager#3638]
## Description

Hello, first of all let me thank you for the amazing job, Voyager is a life saviour with good documentation and very easy to use.

What I would like to know is how can I save/send the images I upload on the Voyager media manager to a git repository, so I don't need to always upload the images for example when cloning the project to a production server.

Hope you can help me with this question.

Best Regards

**Comment by fletch3555** [comment]
You likely wouldn't want to do that.  Imagine you're in production (which should run independently, right?).  Images get uploaded and saved to the filesystem.  Then you go to do another deployment, which has the potential to delete those files (depending how you do it).  Clearly that's not ideal.

Some alternatives you might want to look at are cloud hosting options (AWS has S3 buckets, and DigitalOcean has Spaces, but there are other options as well).

Now, that may not be what you're looking for.  That's implying you want to add images to git through the normal process of running your app.  If, however, you have some images that won't change during the normal run of your app (e.g. "default" avatar images), then all you need to do is add them to the git repo.  If they're in a directory that's already ignored, then you'll have to force it (a flag available to the git CLI).  I'll leave the specifics up to you to research if needed.

tl;dr - Voyager doesn't provide any functionality like that and has no plan to do so in the future.  Also, you probably don't want to do that anyway.
